### ISSUE TITLE
windows ci builds missing dlls

### ISSUE THREAD

**Issue by iedemam** [valiant1x/intensecoinwallet#61]
````
---------------------------
intensecoin-wallet-gui.exe - System Error
---------------------------
The code execution cannot proceed because libgcc_s_seh-1.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
---------------------------
intensecoin-wallet-gui.exe - System Error
---------------------------
The code execution cannot proceed because libwinpthread-1.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
---------------------------
intensecoin-wallet-gui.exe - System Error
---------------------------
The code execution cannot proceed because libstdc++-6.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
---------------------------
intensecoin-wallet-gui.exe - System Error
---------------------------
The code execution cannot proceed because libicuin61.dll was not found. Reinstalling the program may fix this problem. 
---------------------------
OK   
---------------------------
````

**Comment by valiant1x** [comment]
I can confirm this issue is resolved by 68c97d5 / 5a49ec6

**Comment by iedemam** [comment]
👍
